sdk/messaging/azservicebus/session_receiver_test.go

```diff
@@ -132,7 +132,7 @@ func TestSessionReceiver_acceptSessionButAlreadyLocked(t *testing.T) {
 	// You can address a session by name which makes lock contention possible (unlike
 	// messages where the lock token is not a predefined value)
 	receiver, err = client.AcceptSessionForQueue(ctx, queueName, "session-1", nil)
-	require.True(t, internal.IsSessionLockedError(err))
+	require.EqualValues(t, internal.RecoveryKindFatal, internal.ToSBE(context.Background(), err).RecoveryKind)
```

Comment:
At present, the RecoveryKindFatal isn't publicly available.  How does this get exposed up to the caller?

Reply:
There isn't a way for external people to use this. It's purely for internal classification.

I think this properly falls into the case of "the best you can do is log" since the lock itself is taken by some other process/session and you can't stop them.